packages/pretty-format/src/index.js

```diff
@@ -41,6 +41,26 @@ const errorToString = Error.prototype.toString;
 const regExpToString = RegExp.prototype.toString;
 const symbolToString = Symbol.prototype.toString;
 
+// Return whether val is equal to global window object.
+let noWindow;
+let theWindow;
+const isWindow = val => {
+  if (noWindow) {
+    return false;
+  }
+  if (theWindow === undefined) {
+    try {
```

Comment:
Try catch feels like overkill. Why not just `typeof window != 'undefined'`?

Reply:
It catches `ReferenceError: window is not defined` for `--env=node` in Jest or if `pretty-format` package is used for other purpose than testing.

Keep the questions coming. This one caused me to add comments in the code.

Reply:
I added a new comment - it does not.

```sh-session
$ node -p 'typeof window'
undefined
```

Reply:
Or well, the current test does, but that's because the test is referring to `window`, not because it throws from inside pretty-format